### ISSUE TITLE
Click offset for Image detection mode

### ISSUE THREAD

**Issue by byzod** [Nain57/Smart-AutoClicker#484]
### Describe the feature

By default autoclicker can only click predestined position when using image detect mode

But there are many scenarios that a offset is required to click correct position

For example there are many question like this:

```
1. Do you like AAA
O Yes
O No

2. Do you like BBB
O Yes
O No
```
The `O` represents a radio button

If you want to click `Yes` for AAA and `No` for BBB, it's impossible to do so without click offset, captured a image of `O Yes` or `O No` is pointless since all questions have the same answers

### Describe the usage

There're so many many scenario that the indicator part and target part are not the same, it will suit way more situation if the click can have an offset

For a common instance, basically every game have its "buy" button looks the same, you need to check is the price part to decide to buy or not, but you won't click the price, you click the "buy" button under it. 

Klick'r right now has no capability to do this.

### Is it a duplicate?

- [X] There is no duplicates

**Comment by Scr0ll3rz** [comment]
> a really long needed function. you can complicate it a bit, add predefined variables, last match by X and Y, and in the click event add formula input with these variables, LastX+50,LastY-100

How to add predefined variables?

**Comment by SlavaPVR** [comment]
The developer can add and populate these variables when finding an image.

#523

**Comment by Nain57** [comment]
Closing this ticket as the feature have been implemented in 3.0.1-beta01.
Feel free to keep providing any feedback about the feature here
